vercel/resource_project.go

```diff
@@ -105,10 +132,9 @@ At this time you cannot use a Vercel Project resource with in-line ` + "`environ
 						Sensitive:   true,
 					},
 					"id": {
-						Description:   "The ID of the Environment Variable.",
-						Type:          types.StringType,
-						PlanModifiers: tfsdk.AttributePlanModifiers{resource.UseStateForUnknown()},
```

Comment:
We don't need these modifiers anymore? :o

Reply:
It's actually incorrect here - the project resource doesn't bother doing an update on the environment variables, and instead just deletes any that do not match the state after, and adds new ones. 

So we can't rely on stable IDs between updates for env variables.